index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NOVA - PORTFOLIO PROJECT</title>
+    <link rel="stylesheet" href="/assets/style.css">
+</head>
+<body>
+
+    <section class="start">
+        <header class="navbar">
+            <nav class="navbar__container">
+                <img class="logo" src="/images/logo.svg" alt="nova logotype">
+                <ul class="navbar__list">
+
+                    <li class="navbar__element">
+                        <a href="#team" class="navbar__link">The Team</a>
+                    </li>
+
+                    <li class="navbar__element">
+                        <a href="#pricing" class="navbar__link">Pricing</a>
+                    </li>
+
+                    <li class="navbar__element">
+                        <a href="#features" class="navbar__link">Features</a>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+
+        <div class="hero">
+            <div class="hero__container">
+
+                <div class="hero__column">
+                    <h1 class="hero__title">Beautiful Free Nova template</h1>
+                    <p class="hero__description">A top notch premium quality template for your next web project.</p>
+                    <button class="hero__button">Download FREE!</button>
+                </div>
+
+                <div class="hero__column--wider">
+                    <img src="/images/screen.png" alt="screen">
+                </div>
+
+            </div>
+        </div>
+    </section>
+
+    <section class="features" id="features">
+        <h2 class="features__title">FEATURES</h2>
+        <div class="features__container">
+
+            <div class="features__element element">
```

Comment:
Zastanawim się czy bym tutaj nie zrobił `ul > li`, bo mamy "listę" cech czyli "...container" to `ul`, a "element" to `li`